TrajectoryPlannerROS:
   max_vel_x: 0.35
   min_vel_x: 0.02
   # max_vel_y: 0.0  # zero for a differential drive robot
   # min_vel_y: 0.0
   # max_vel_theta: 2.0
   # min_vel_theta: -2.0  # Setting this negative is critical or the robot will only turn one way
   # min_in_place_vel_theta: 0.2
   # min_in_place_rotational_vel: 0.2     # old parameter name
   # escape_vel: -0.05
   # acc_lim_x:                    2.5 
   # acc_lim_y:                    0.0    # 0 for non-holonomic robot
   # acc_lim_theta:                0.1

   holonomic_robot: false
   meter_scoring: true
   yaw_goal_tolerance:        2.0  
   xy_goal_tolerance:         0.8  

